scripts/unit.sh

```diff
@@ -2,9 +2,4 @@
 
 set -eu
 
-HERE="$(dirname "$(readlink -f "$BASH_SOURCE")")"
-REPO_ROOT="$(realpath "${HERE}/..")"
-
-cd "${HERE}"
```

Comment:
You deleted this in a lot of places but I think it will mean that these scripts can't be run from the `scripts` dir now?

Reply:
Oh, I see. I can revert this, if you want